libs/editor/src/lib/services/wizard.service.ts

```diff
@@ -13,8 +13,25 @@ export class WizardService {
   private wizardStep = 1
   private wizardData: Map<string, any> = new Map()
 
-  constructor(private translateService: TranslateService) {
-    this.initialize()
+  private id: string
+
+  constructor(private translateService: TranslateService) {}
+
+  initialize(id: string): void {
+    this.id = id
+    this.wizardData.clear()
+
+    const savedWizardData = localStorage.getItem(id)
```

Comment:
almost, the id of the batch should not be ids of the localStorage, but ids of the dictionnary stored in the localStorage
```js
localStorage
 - datafeeder-state: {
     d7c14181-2e66-49ff-868e-8c55671f955f: [{},{}]...,     
     trtrez543534543543534: [{},{}]...,
}
```